config/yamllint.yaml

```diff
@@ -0,0 +1,37 @@
+---
+
+yaml-files:
+  - '*.yaml'
+  - '*.yml'
+
+rules:
+  braces: enable
+  brackets:
+    min-spaces-inside: 1
+    max-spaces-inside: 3
```

Comment:
What is this? Is this inclusive or exclusive?

Reply:
Inclusive, I reduced this to 1/1 to enforce `[ 'value' ]` style.